source/Calamari.Tests/Fixtures/Certificates/ImportCertificateCommandFixture.cs

```diff
@@ -1,18 +1,19 @@
-﻿#if WINDOWS_CERTIFICATE_STORE_SUPPORT
-using System;
+﻿using System;
 using System.IO;
 using System.Security.Cryptography.X509Certificates;
 using Calamari.Common.Plumbing.FileSystem;
 using Calamari.Common.Plumbing.Variables;
 using Calamari.Deployment;
 using Calamari.Testing.Helpers;
+using Calamari.Testing.Requirements;
 using Calamari.Tests.Helpers;
 using Calamari.Tests.Helpers.Certificates;
 using NUnit.Framework;
 using Octostache;
 
 namespace Calamari.Tests.Fixtures.Certificates
 {
+    [Category(TestCategory.CompatibleOS.OnlyWindows)]
```

Comment:
Previously this whole test wouldnt even exist in netcore, now netcore can run some of these tests so long as its on windows